tests/cephfs/cephfs_upgrade/upgrade_pre_req.py

```diff
@@ -116,73 +125,145 @@ def run(ceph_cluster, **kw):
+                    f"{nfs_export_name} {fs_name} path={path}",
+                )
+
             # Verify ceph nfs export is created
-            out, rc = clients[0].exec_command(
+            out, rc = nfs_client[0].exec_command(
                 sudo=True, cmd=f"ceph nfs export ls {nfs_name}"
             )
-            output = out.read().decode()
-            output.split()
-            if nfs_export_name in output:
+            if nfs_export_name in out:
                 log.info("ceph nfs export created successfully")
             else:
                 raise CommandFailed("Failed to create nfs export")
             # Mount ceph nfs exports
-            clients[0].exec_command(sudo=True, cmd=f"mkdir -p {nfs_mounting_dir}")
+            nfs_client[0].exec_command(sudo=True, cmd=f"mkdir -p {nfs_mounting_dir}")
             assert fs_util.wait_for_cmd_to_succeed(
-                clients[0],
-                cmd=f"mount -t nfs -o port=2049 {nfs_server}:{nfs_export_name} {nfs_mounting_dir}",
+                nfs_client[0],
+                cmd=f"mount -t nfs -o port=2049 {nfs_server_name}:{nfs_export_name} {nfs_mounting_dir}",
             )
-            clients[0].exec_command(
+            nfs_client[0].exec_command(
                 sudo=True,
-                cmd=f"mount -t nfs -o port=2049 {nfs_server}:{nfs_export_name} {nfs_mounting_dir}",
+                cmd=f"mount -t nfs -o port=2049 {nfs_server_name}:{nfs_export_name} {nfs_mounting_dir}",
             )
-            out, rc = clients[0].exec_command(cmd="mount")
-            mount_output = out.read().decode()
-            mount_output = mount_output.split()
+            out, rc = nfs_client[0].exec_command(cmd="mount")
+            mount_output = out.split()
             log.info("Checking if nfs mount is is passed of failed:")
             assert nfs_mounting_dir.rstrip("/") in mount_output
             log.info("Creating Directory")
-            out, rc = clients[0].exec_command(
+            out, rc = nfs_client[0].exec_command(
                 sudo=True, cmd=f"mkdir {nfs_mounting_dir}{dir_name}"
             )
+            nfs_client[0].exec_command(
+                sudo=True,
+                cmd=f"python3 /home/cephuser/smallfile/smallfile_cli.py --operation create --threads 10 --file-size 4 "
+                f"--files 1000 --files-per-dir 10 --dirs-per-dir 2 --top "
+                f"{nfs_mounting_dir}{dir_name}",
+                long_running=True,
+            )
+            nfs_client[0].exec_command(
+                sudo=True,
+                cmd=f"python3 /home/cephuser/smallfile/smallfile_cli.py --operation read --threads 10 --file-size 4 "
+                f"--files 1000 --files-per-dir 10 --dirs-per-dir 2 --top "
+                f"{nfs_mounting_dir}{dir_name}",
+                long_running=True,
+            )
+
+        else:
+
+            clients = ceph_cluster.get_ceph_objects("client")
```

Comment:
I am not seeing this nfs block getting executed in logs. Could you check that once?

Reply:
Just one thing, I am not seeing ceph-fuse mounts in latest logs.